src/main/java/ladder/domain/Position.java

```diff
@@ -0,0 +1,60 @@
+package ladder.domain;
+
+import ladder.domain.exception.LessThanMinimumNumberException;
+import ladder.domain.exception.NonNumericStringException;
+
+import java.util.Objects;
+
+public class Position {
+
+    public static final int MINIMUM = 0;
+
+    private final int value;
+
+    public Position() {
+        this.value = 0;
```

Comment:
주(main) 생성자를 호출하는 방식으로 바꿔보면 어떨까요?
```suggestion
        this(MINIMUM);
```

Reply:
주 생성자를 사용하게 되면 중복되는 코드가 줄어들수 있고 실수를 할 확률도 줄어들 것 같네요. 혹시 다른 장점이 더 있을까요?

Reply:
말씀해주신대로 유지보수적인 측면에서 장점이 크다고 볼 수 있을 것 같아요 👍